website/source/docs/enterprise/runs/variables-and-configuration.html.md

```diff
@@ -75,9 +80,42 @@ For any of the `GITHUB_` attributes, the value of the environment variable will
 be the empty string (`""`) if the resource is not connected to GitHub or if the
 resource was created outside of GitHub (like using `terraform push`).
 
+### Personal Variables
+
+Personal variables can be created at the Environment level. All Personal
+Environment variables are private and scoped to the user that created them. Just
+like Environment variables, they are injected into the virtual environment
+during the `plan` and `apply` phases.
+
+Personal Environment variables can be used to override Environment variables on
+a per-user basis. You can add, edit, and delete environment variables from the
+"variables" page on your environment.
+
+## Variable Hierarchy 
+
+For Environment and Personal Environment variables, it is possible to create the
+same variable in multiple places. Variables are applied in the following order:
+
+1. Environment
+2. Personal Environment
+
+Here's an example: 
+
+* For the `SlothCorp/petting_zoo` environment, a user creates
+an Environment variable called `SECRET_GATE_ACCESS_KEY` and sets the value to
+`"orange-turtleneck"`
+* Another user adds a Personal Environment variable for 
+`SECRET_GATE_ACCESS_KEY` and sets the value to `"pink-turtleneck"`
+* When the `plan` or `apply` executes, the `SECRET_GATE_ACCESS_KEY` will use
+`"pink-turtleneck"`
+
```

Comment:
* When any other user submits a `plan` or `apply`, the `SECRET_GATE_ACCESS_KEY` will use `orange-turtleneck`